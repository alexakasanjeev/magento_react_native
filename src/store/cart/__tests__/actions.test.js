import {
  createQuoteId,
  getCustomerCart,
  getCartItemProduct,
  removeItemFromCart,
} from '../actions';

describe('cart actions', () => {
  test('createQuoteId()', () => {
    // Exercise
    const result = createQuoteId();

    // Verify
    expect(result).toMatchSnapshot();
  });

  test('getCustomerCart()', () => {
    // Exercise
    const result = getCustomerCart();

    // Verify
    expect(result).toMatchSnapshot();
  });

  test('getCartItemProduct()', () => {
    // Setup
    const sku = 'WH02-S';

    // Exercise
    const result = getCartItemProduct(sku);

    // Verify
    expect(result).toMatchSnapshot();
  });

  test('removeItemFromCart()', () => {
    // Setup
    const itemId = 'WH02-S';

    // Exercise
    const result = removeItemFromCart(itemId);

    // Verify
    expect(result).toMatchSnapshot();
  });
});
