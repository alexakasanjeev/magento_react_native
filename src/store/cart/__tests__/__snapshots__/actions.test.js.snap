// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cart actions createQuoteId() 1`] = `
Object {
  "type": "MAGENTO_CREATE_QUOTE_ID_REQUEST",
}
`;

exports[`cart actions getCartItemProduct() 1`] = `
Object {
  "payload": "WH02-S",
  "type": "MAGENTO_CART_ITEM_PRODUCT_REQUEST",
}
`;

exports[`cart actions getCustomerCart() 1`] = `
Object {
  "type": "MAGENTO_CUSTOMER_CART_REQUEST",
}
`;

exports[`cart actions removeItemFromCart() 1`] = `
Object {
  "payload": Object {
    "itemId": "WH02-S",
  },
  "type": "MAGENTO_REMOVE_ITEM_FROM_CART_REQUEST",
}
`;
